### DISCUSSION TITLE
What is the relationship between "Wallet 1 Addresses" and the new receiving address in the Chia blockchain light wallet interface?

### DISCUSSION THREAD

**Discussion by RedChimae** [Chia-Network/chia-blockchain#18481] [Support]
I have a question that needs clarification. As shown in the diagram, in the Chia blockchain light wallet interface, what is the relationship between the Wallet 1 Addresses indicated by the white arrow and the new receive address feature? Why is my Wallet 1 Addresses (which seems like a form of contact or identification) fixed and unchanging, while the "New Receive Address" button above it can generate new receiving addresses randomly? If I want to withdraw funds from Hpool mining pool to my Chia blockchain light wallet, which address should I use as the recipient address? Is it the fixed Wallet 1 Addresses?
 
![捕获 Chia blockchain 轻钱包 联系人](https://github.com/user-attachments/assets/f23e2815-95ff-4e3f-af94-10b7d74d7fca)

**Comment by BrandtH22** [comment]
Hey @RedChimae , as a heads up we can generally provide more timely, thorough support in our discord server (https://discord.gg/chia).

As for receive addresses, your wallet can have a nearly infinite number of xch addresses. That wallet 1 is the first xch address associated with your wallet but for various reasons new xch addresses are created (like receiving change from a transaction or cancelling an offer). The most recent xch address that has been generated will be displayed in the top receive address section. 

Any of the xch addresses will work as the receive address for the pool.
What I personally did is have a new xch address generated and added that to my address book as the pool receive address then I generated another new address to ensure I only use that pool address for my pool rewards. This allows me to just look up that pool receive address in a block explorer to see exactly how much xch I have received in rewards from the specific pool. This is just my personal method of keeping the accounting side of things sane but in summation you can use either the wallet 1 or the address in the receive address section.

Let us know if any other questions arise 👍

**Reply by RedChimae** [reply]
Thank you for the reminder! I haven't tried interacting on the Discord server (https://discord.gg/chia) yet because I'm used to BBS-style forums.

I understand your previous answer. Here are a few more questions I'd like to ask:

1. If I click on the new receiving address at the top, and the address gets updated, does the previous address still remain valid?
2. What does the verifiable credential that resembles an ID card represent, and how do I use it? Does Chia have an official manual explaining this?
3. What does the term "offer" mean, and how is it used? Can I use it to sell Chia tokens from my wallet to others, and would my earnings be in real currency?
4. Why can't I see the full node after switching the Chia blockchain working mode to light wallet? Does the light wallet mode not require synchronization with full nodes?

**Reply by OverActiveBladderSystem** [reply]
1. Your "keys" (the 24 words, never share with anyone) for each wallet can create many receive addresses, the addresses are tied to those keys and all remain valid even when you use a new address for that wallet. The intention of new receive addresses for each transaction was to provide some obscurity between transactions (You can reuse only one address if you want, thats an option for advanced users to change in the config.yaml however not the GUI).

You can read more about addresses and the light wallet here https://docs.chia.net/faq/#how-does-light-wallet-syncing-work

2. The VC/DID creates a way for you to digitally identify yourself and ownership of assets on chain. Lets say you create an NFT collection which you assign your DID to, now people know your DID created that asset and its not someone imitating you, this also opens up scenarios where only people you authorize are able to interact with certain assets in certain ways. Think of it like a digital passport to verify your identity.

You can read more about Verifiable Credentials https://docs.chia.net/guides/verifiable-credentials-guide/#intro
and DIDs here https://docs.chia.net/academy-did/

3. Offers are partially signed transactions, a second party must fulfill the requirements of the offer you create. Lets say I create an "offer" file to trade my NFT for 1 XCH, I share that offer file on social media websites (you can post it anywhere), people can view the details of the offer and decide if they want to complete and submit it to the network, no 3rd party is required to host the assets from either side (aka you eliminate the need for middle men brokers between the two parties) just "The Maker and the Taker" are needed. This does not work with assets off chain (currency), only assets that are on chain (tokenized currency).

You can read more about offers here. https://docs.chia.net/academy-offers/

4. In "wallet mode" you will only run the wallet service and it will ask several random full nodes on the network for data your wallet needs to sync, so that you can send transactions and confirm any incoming transactions. This lets you bypass storing the full blockchain as the farming mode does (Currently the blockchain database is 175GB), farmers ("miners") storing the blockchain allows them to act independent of traditional crypto pools that control things like the mempool and blockchain state.

If you prefer to run your own local full node for your wallet, you can use farming mode and just ignore the other services.

You can read more about switching modes here https://docs.chia.net/getting-started/wallet-guide/#run-and-configure-chia

**Comment by RedChimae** [comment]
Continue asking questions

**Comment by BrandtH22** [comment]
Closed due to no further questions from OP, please reopen this ticket or open a new one if the issue or questions persist
